.github/workflows/link-check-exclude.yml

```diff
@@ -0,0 +1,27 @@
+name: Link Check Exclude
+on: deployment
```

Comment:
hmm ... why on deployment?

Reply:
That's where it was in the CircleCI config, unless I misread it

Reply:
Here's the bit I'm referring to:

```
jobs:
  test:
    <<: *defaults
    steps:
      - install
      - run: yarn test
      - run: yarn lint-css
      - run: yarn lint-ts
      - run: yarn format-check
      - run: yarn link-check-diff
      - run: yarn link-check-exclude

  test_full:
    <<: *defaults
    steps:
      - install
      - run: yarn test
      - run: yarn lint-css
      - run: yarn lint-ts
      - run: yarn format-check
      - run: yarn link-check
      - run: yarn link-check-exclude

workflows:
  version: 2

  commit:
    jobs:
      - test
```

`test` includes `link-check-exclude`, and `test` is run on `commit`. If you're implying that it would make more sense to run this as part of the daily check, I agree and can make that change.